### ISSUE TITLE
Error while calling lua chunk

### ISSUE THREAD

**Issue by qrockz** [EdenEast/nightfox.nvim#275]
## Description

After successfully installing the plugin with the plugin manager vimplug and enabling the nightfox color scheme with the command `colorscheme nightfox`
i get the following error msg in neovim:

> .local/share/nvim/site/plugged/nightfox.nvim/colors/nightfox.vim:                                      
> line    9:
> E5105: Error while calling lua chunk: [string "<VimL compiled string>"]:2: attempt to index field 'g' (a nil value)

specs:

> 5.4.0-132-generic #148-Ubuntu SMP Mon Oct 17 16:02:06 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
> NVIM v0.4.3
> Build type: Release
> LuaJIT 2.1.0-beta3
> Lua 5.2.4  Copyright (C) 1994-2015 Lua.org, PUC-Rio
> GNOME Terminal 3.36.2

init.vim:
```
 call plug#begin('~/.local/share/nvim/site/plugged')
     Plug 'EdenEast/nightfox.nvim' 
     Plug 'morhetz/gruvbox'
 call plug#end()
 colorscheme nightfox 
```

Do i miss something (other colorschemes like gruvbox is working)?

**Comment by EdenEast** [comment]
It looks like `vim.g` does not exists. This was added when lua was added as a target language in neovim `0.5.0`. Nightfox supports the current released version of neovim version `0.8.0`. See the [requirements](https://github.com/EdenEast/nightfox.nvim#requirements) section for more information.

**Comment by qrockz** [comment]
Thx, You were right!
`sudo apt install nvim` has installed an older version of nvim which is indeed not working with your plugin it seems.

Now I have installed the latest release version of nvim (0.8.1) from the official repo (as an appimage) I get no errors when loading your colorscheme.

Thx for your lightning fast response!
